cmake/SelectHashes.cmake

```
@@ -56,6 +56,8 @@ ELSE()
 	MESSAGE(FATAL_ERROR "Asked for unknown SHA1 backend: ${USE_SHA1}")
 ENDIF()
 
+FILE(GLOB SRC_SHA1_UMBRELLA "hash/sha1.h")
+list(APPEND SRC_SHA1 ${SRC_SHA1_UMBRELLA})
```

I don't think that we need a variable here - I think that this should just be:

```suggestion
FILE(GLOB SRC_SHA1_UMBRELLA "hash/sha1.h")
list(APPEND SRC_SHA1 "hash/sha1.h")
```